README.md

```diff
@@ -50,104 +48,106 @@ Make the CI green again.
-| 2018-10-31 13:28 | 0       | 40      | 13       | 4       | 43      | 41.67%     |
-| 2018-11-19 10:32 | 0       | 48      | 8        | 5       | 39      | 50.53%     |
-| 2018-12-08 20:37 | 2       | 18      | 4        | 3       | 73      | 18.95%     |
-
-## Handling Failed CI runs
-
-### Flaky Tests
-
-TODO: automate all of this in ncu-ci
-
-#### Identifying Flaky Tests
-
-When checking the CI results of a PR, if there is one or more failed tests (with
-`not ok` as the TAP result):
-
-1.  If the failed test is not related to the PR (does not touch the modified
-    code path), search the test name in the issue tracker of this repo. If there
-    is an existing issue, add a reply there using the [reproduction template](./templates/repro.txt),
-    and open a pull request updating `flakes.json`.
-2.  If there are no new existing issues about the test, run the CI again. If the
-    failure disappears in the next run, then it is potential flake. See
-    [When discovering a potential flake on the CI](#when-discovering-a-potential-new-flake-on-the-ci)
-    on what to do for a new flake.
-3.  If the failure reproduces in the next run, it is likely that the failure is
-    related to the PR. Do not re-run CI without code changes in the next 24
-    hours, try to debug the failure.
-4.  If the cause of the failure still cannot be identified 24 hours later, and
-    the code has not been changed, start a CI run and see if the failure
-    disappears. Go back to step 3 if the failure still reproduces, and go to
-    step 2 if the failure disappears.
-
-#### When Discovering a Potential New Flake on the CI
-
-1.  Open an issue in this repo using [the flake issue template](./templates/flake.txt):
+[A GitHub workflow](.github/workflows/reliability_report.yml) is run every day
+to produce reliability reports of the `node-test-pull-request` CI and post
+it to [the issue tracker](https://github.com/nodejs/reliability/issues).
+
+## Protocols in improving CI reliability
+
+Most work starts with opening the issue tracker of this repository and
+reading the latest report. If the report is missing, see
+[the actions page](https://github.com/nodejs/reliability/actions) for
+details. GitHub's API restricts the length of issue messages, so
+whenever the report is too long the workflow can fail to post the
+issue. But it should still leave a summary in the actions page.
+
+### Identifying flaky JS tests
+
+1. Check out the `JSTest Failure` section of the latest reliability report.
+    It contains information about the JS tests that failed more than 1 pull
+    requests in the last 100 `node-test-pull-request` CI runs. The more
+    pull requests a test fail, the higher it would be ranked, and the more
+    likely that it is a flake.
+2. Search the name of the test in [the Node.js issue tracker](https://github.com/nodejs/node/issues)
+    and see if there is already an issue about it. If there is already
```

Comment:
identation? also same under 1. above as well.

Reply:
Updated to use two-space intetation